org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/test/utils/CompareUtilities.java

```diff
@@ -361,6 +397,14 @@ private static boolean matches(String actualJsonString, String expectedJsonStrin
           }
         }
         return true;
+      } else if (expectedJsonString.startsWith("$external:")) {
+        String[] cmd = expectedJsonString.substring(1, expectedJsonString.length() - 1).split("\\:");
```

Comment:
Same issue as above, ":" shouldn't be escaped

Reply:
same response !